pkg/controller/webhook/webhook.go

```diff
@@ -89,6 +108,41 @@ func notify(destination string, message []byte) error {
 		destination, resp.Status)
 }
 
+func notifyWS(destination string, message []byte) error {
+	ctx, cancel := context.WithTimeout(context.Background(), notificationSendTimeout)
+	defer cancel()
+
+	c, rs, err := websocket.Dial(ctx, destination, nil)
```

Comment:
So this is useful variation where the REST server dials a WebSocket...

But we really need a version where the REST server exposes a WebSocket and the client dials it. This way the client does not need an exposed endpoint.

Reply:
> The implementation should work the same as it does in ACA-Py. The WebSocket support will allow for clients to register dynamically after the server has started. The full duplex functionality of WebSocket will not be used - it should be a one-way notification from server->client. The endpoint will be /ws.

Reply:
Will add that feature as part of separate PR